File: pgm/ABC_Java/ABC_260_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String S[] = reader.readLine().split ("");
        final int no = -1;
        final int one = 1;
        final int two = 2;
        final int three = 3;
        boolean uniqueFlg = false;
        for (int i = 0; i < S.length; i++) {
            if (uniqueFlg)
                break;
            if (!S[i].equals (S[(i + one) % three]) &&
                !S[i].equals (S[(i + two) % three])) {
                uniqueFlg = true;
                System.out.println (S[i]);
            }
        }
        if (!uniqueFlg)
            System.out.println (no);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_260_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.*
// import java.util.Scanner
// import java.util.StringTokenizer

// import kotlin.math.*

const val one: Int = 1

fun main() {
    val S: String = readln()
    println(S.groupingBy { it: Char -> it }
        .eachCount()
        .filterValues { cnt: Int -> cnt == one }
        .keys
        .firstOrNull() ?: -one)
    kotlin.system.exitProcess(0)
}